content/10.diagnostics.md

```diff
@@ -100,9 +117,7 @@ Specifically, the individuals at risk include:
 - people with severe symptoms
 - people showing mild symptoms that have been in contact with a person who has developed the infection
 - people with underlying health conditions
-- pregnant women
 
-However, this method of testing administration does not detect a high proportion of infections.
-This includes individuals that are asymptomatic (potentially able to infect others) and individuals that are able to recover at home.
-Furthermore, individuals for whom the origin of the infection is uncertain are not tested.
+However, this method of testing administration does not detect a high proportion of infections, and does not allow for test-and-trace methods to be used.
+Individuals that are asymptomatic (potentially able to infect others) and individuals that are able to recover at home are therefore often unaware of their status.
```

Comment:
"that": who
"(potentially able to infect others)": Either explain in a separate sentence or too self-explanatory to mention.

"that are able to recover at home": Is it really the attribute that they are able recover at home that makes them unaware of their status? I think it's more a psychological phenomenon to wait things out until they are too worse to neglect. Who isn't able to recover at home anyways?

Reply:
Perhaps "who are not advised to seek in-person medical attention" or something? Right now I believe the CDC recommendation is to stay at home and call your doctor rather than going in for an appointment until you get very severe. There is supposed to be drive through testing, but my understanding is that it is still not very accessible (all based on the USA, it's definitely better in some other places!)

Reply:
This topic might be more complex than one sentence can explain. And it's also an interesting and useful topic to elaborate. Unidentified cases drive the spread, after all. It will most likely depend on the country, its policies, and the socio-cultural environment. Here are some aspects I spontaneously figured:
* Insufficient Capacities: What are the testing capacities? How well is the medical system equipped? How well can contact tracing be applied?
* Incomplete and inaccurate case identification: Is everyone allowed to be tested? Several countries only test cases with a traceable infection history. Some countries only test severe symptoms. Different testing methods are used.
* Financial insecurity: Is the test being paid for? Is follow-up health care being paid for? Does the government compensate for the missed wages?
* Improper sensitization: How do the government, local scientists, and the press sensitize the topic? A downplayed risk assessment can desensitize people to not take it serious enough. They play it safe, neglect it, might even spread it because they still think "it's just the flu" or because they start to believe some of the "conspiracy theories".

Reply:
Sorry I missed this before, @matfax! It is very interesting! I don't know how much public health we have/will have in the text, but these are all really interesting points.